### ISSUE TITLE
After Win 11 March Update 2023 , TranslucentTB dont working [Bug]: 

### ISSUE THREAD

**Issue by Svencore** [TranslucentTB/TranslucentTB#741]
### Please check for a duplicate first

- [X] I have checked the issue tracker if my bug has already been fixed or answered in another (possibly closed) issue.

### Steps to reproduce the issue

Deinstallation, new Installation, Reboot dont fix this issue.TranslucentTB dont working, Deinstallation the Last Update from Win11 (01.03.2023) fix the issue.

### Expected results

Translucent TB dont work with new Win11 Update 

### Actual results

App without any function 

### Windows version

Windows 11 RTM (build 22000)

**Comment by pittmarl** [comment]
Concur.
Confirmed.
Microsoft Store neutered Application - does nothing.
Directly after the Tuesday update.
Happening within multiple machines on premise.

**Comment by TheRealDjElite** [comment]
I am on February 28, 2023—KB5022913 (OS Build 22621.1344), and can confirm this issue.

**Comment by pittmarl** [comment]
Update Today, March 01, 2023
KB4023057: Update for Windows Update Service components

No impact, application still neutered.

**Comment by Svencore** [comment]
Hope for an Update for Translucent TB to fix it, a very useful and pretty App

**Comment by Tech127x** [comment]
Confirmed on 22621.1344
This app is extremely important to me.  Please update asap!  Thanks!!

**Comment by ZamenisL** [comment]
Same here, build 22621.1344

**Comment by sylveon** [comment]
Duplicate of #537

**Comment by Svencore** [comment]
Okay, i have dont see this Post, sorry!

**Comment by Alej0ff** [comment]
> Coincidir. Empedernido. Aplicación castrada de Microsoft Store: no hace nada. Directamente después de la actualización del martes. Sucediendo dentro de múltiples máquinas en las instalaciones.
bro, do you still have the problem with translucent?
Because I have the February 28 update and it doesn't work for me

**Comment by PingusDingus** [comment]
I had experienced the same issue, safe to say this problem is not uncommon. But I followed the instructions some dude left on another thread and now it's fixed.

Credits to **PsychoPops** on github quoting


"Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows, reboot your computer and your task bar should be transparent again. Thats what worked for me."

This worked for me with no issues so far been a month now
